components/post_view/post_message_view/post_message_view.jsx

```diff
@@ -208,32 +236,55 @@ export default class PostMessageView extends React.PureComponent {
         const mentionKeys = [...this.props.mentionKeys, this.props.currentUser.username];
 
         const options = Object.assign({}, this.props.options, {
-            emojis: this.props.emojis,
-            siteURL: this.props.siteUrl,
+            emojis,
+            siteURL: siteUrl,
```

Comment:
This is just asking for trouble. I think we had a plan to standardize these though, so it's fine for now

Reply:
Noted, I'll do the clean up related to this. I'll post on dev channel on which should be included (e.g. `...Html or ...HTML`).

Reply:
"HTML" imo.

I'm like 4/5 that it should be "HTML" for Go code, just because that's a well-established convention for Go. For JavaScript, maybe the convention isn't quite as well established, but I do think "HTML" is still more common for web ([for example](https://developer.mozilla.org/en-US/docs/Web/API)), and it'd be nice if JavaScript and Go used the same convention.